### PR TITLE
drop versioneer and use setuptools_scm instead

### PR DESCRIPTION
**Pull request by ocefpaf**
@Conengmo can you review this one and merge before #1409?

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -53,9 +53,8 @@
 
 # The full version, including alpha/beta/rc tags.
 
-from folium._version import get_versions
-release = get_versions()['version']
-del get_versions
+import folium
```
[the remark this answers is not in the document]
Comment (ocefpaf) [reply docs/conf.py:56]: Let's ignore this one. It is better to have that import near where it is used in this context.

File: MANIFEST.in [hunk 2]
```diff
@@ -4,5 +4,6 @@ recursive-include folium *.py
 recursive-include folium *.js
 recursive-include folium/plugins *
 recursive-include folium/templates *
-include versioneer.py
+
 prune examples/
+exclude foium/_version.py
```
Comment (Conengmo) [inline MANIFEST.in:9]:
```suggestion
exclude folium/_version.py
```

File: setup.py [hunk 3]
```diff
@@ -92,5 +89,9 @@ def walk_subpkg(name):
     extras_require={"testing": ["pytest"]},
     install_requires=install_requires,
     zip_safe=False,
-    cmdclass=versioneer.get_cmdclass(),
+    use_scm_version={
```
Comment (Conengmo) [inline setup.py:92]: I haven't used scm before, so I can't really comment on this. I've been reading some docs, and they say to add `setup_requires=['setuptools_scm'],` here as well. https://github.com/pypa/setuptools_scm/#setuppy-usage If you're sure that this works as is then let's go ahead.

Comment (ocefpaf) [reply setup.py:92]: That is correct but my next PR would remove it b/c I'm planning on adding a pyproject.toml to make folium pep 517/518 compliant.

Comment (Conengmo) [reply setup.py:92]: Sounds good, so this is merely an intermediate change. Go ahead then.

**Pull request merged**